src/app/_components/MainPageSections/GoalSection.js

```diff
@@ -0,0 +1,12 @@
+import { Paragraph } from '@components/Typography';
+
+export function GoalSection() {
+  return (
+    <section className="my-[24px] flex w-screen flex-col items-center gap-4 px-4 py-4 lg:my-[0px] lg:gap-6 lg:px-0 lg:py-32">
+      <Paragraph className="text-center text-p2 font-bold text-secondary-400 lg:text-h3">Наша ціль</Paragraph>
```

Comment:
Just a suggestion to use `<Heading type="h3 (or h2)" /> ` instead of `<Paragraph />`

Reply:
Thanks for comment! If we'll use `Heading` we need to change text parameters to `font-size: 1.125rem/* 18px */;
  line-height: 1.75rem/* 28px */;`, we don't have such values in heading font system, so we need to use extra code to choose between components. I thinks its better just to leave one of them(`Paragraph` or  `Heading`) not really matters.

Reply:
But we can just add className="text-p2" to the heading component, can't we?

Reply:
Do you mean to use ` <Heading className="text-center text-p2 font-bold text-secondary-400 lg:text-h3">Наша ціль</Heading>` instead of Paragraph block of code?